build-tools/helm/spark-kubernetes-operator/crds/sparkapplications.org.apache.spark-v1.yml

```diff
@@ -0,0 +1,6947 @@
+# Generated by Fabric8 CRDGenerator, manual edits might get overwritten!
```

Comment:
I'm wondering if we can avoid having this by generating this during the compile time.

Reply:
Yes, this can be achieved. It's added for 

* reference
* packed & used by the helm chart

But this is not mandatory. We already have a gradle task to automatically generate this at compile time.

If it's more preferred to not have the generated yaml checked in, I can remove it when implementing the chart publishing to helm reporistory.

Reply:
if it's easy(without much effort to set up external building tools, especially OS coupled) to generate the constant code, I think we can remove it, just like we did for antlr, grpc in the spark main repo

Reply:
Should be straightforward - removed this https://github.com/apache/spark-kubernetes-operator/pull/2/commits/39a74e19604f08698f19853def37f08142a023be

docs at  https://github.com/apache/spark-kubernetes-operator/pull/2/commits/d4480921b72331e4d835d43c35597d9af7e11006